Geometry.cs

```diff
@@ -30,7 +30,11 @@ namespace GrayBMP;
    /// <summary>Returns angle between two points.</summary>
    public readonly double AngleTo (Point2 b) {
       double dx = b.X - X, dy = b.Y - Y;
-      return Math.Atan2 (dy, dx);
+      if (dx != 0 || dy != 0) {
+         var fAngle = Math.Atan2 (dy, dx);
+         return fAngle.EQ (-Math.PI) ? Math.PI : fAngle;

```

Comment:
Why this special treatment for -PI ?

Reply:
When delta Y is negative zero (applicable for floating point numbers), we might get -180 degrees. Although this piece of code is not applicable in this case, I just added this for completeness of the **AngleTo** method.